donkeycar/parts/actuator.py

```diff
@@ -805,3 +823,31 @@ def shutdown(self):
         # stop vehicle
         self.run(0)
         self.running = False
+
+
+class SunFounder:
+    # to use this actuator change import SunFounder in manage.py, change throttle to SunFounder(),
+    # and comment out throttle_controller
+    def __init__(self):
+        import logging        
+        import picar
+        from picar import back_wheels
+        logger = logging.getLogger(__name__)
+        logger.info('Set up back_wheels')        
+        picar.setup()
+        # place path to your car's config file below
+        db_file = "/home/pi/SunFounder_PiCar-V/remote_control/remote_control/driver/config"
```

Comment:
This should be passed as an argument, and then we should have two config parameters in `cfg_complete.py`:
```python
SUNFOUNDER = False
SUNFOUNDER_RC_CFG_PATH = "/home/pi/SunFounder_PiCar-V/remote_control/remote_control/driver/config"
```
Add an entry in `complete.py` which does what you wrote in the comment above, i.e. change the throttle controller to your new part. Then it should work for the SunFounder car without tweaking any code, just an update of the `myconfig.py` file. 

You should probably also write a comment in the class how to install the `picar` package.